.github/workflows/main_win.yaml

```diff
@@ -88,6 +88,11 @@ jobs:
         run: echo "docker_service=$(cat ${GITHUB_WORKSPACE}/.github/config/${{ inputs.platform }}.json | jq -rc '.docker_service')" >> $GITHUB_ENV
       - id: set-docker-runner-service
         run: echo "docker_runner_service=$(cat ${GITHUB_WORKSPACE}/.github/config/${{ inputs.platform }}.json | jq -rc '.docker_runner_service')" >> $GITHUB_ENV
+      - uses: actions/upload-artifact@v3
```

Comment:
I don't know that you need the change it here, but wasn't there a discussion about needing to use @git hashes here rather than tags ? Or does that apply to actions outside of `actions/` only ?

Reply:
We don't need to use hash and pin github owned actions.